src/admin/admin_classauction_tools.php

```diff
@@ -2,78 +2,6 @@
 
 class Admin_classauction_tools extends Admin_site
 {
-    /**
-     * @deprecated see if still used, if not, remove
-     */
-    function display_admin_tools_license()
-    {
-        if (isset($_GET['licenseRefreshed']) && $_GET['licenseRefreshed']) {
-            geoAdmin::m('License Data successfully refreshed from license server.', geoAdmin::SUCCESS);
-        }
-
-        $view = geoView::getInstance();
-
-        $leased = geoPC::is_leased();
-
-        $tpl_vars = array();
-        if (defined('DEMO_MODE')) {
-            $key = 'DEMO';
-        } elseif (defined('DEMO_MODE_TEXT')) {
-            $key = DEMO_MODE_TEXT;
-        } else {
-            $key = $this->db->get_site_setting('license');
-        }
-        $tpl_vars['licenseKey'] = $key;
-        $tpl_vars['leased'] = $leased;
-        $tpl_vars['force_powered_by'] = geoPC::force_powered_by();
-        $tpl_vars['trial'] = geoPC::is_trial();
-        $tpl_vars['demo'] = defined('DEMO_MODE');
-        $tpl_vars['license_only'] = geoPC::license_only();
-        $tpl_vars['white_label'] = geoPC::is_whitelabel();
-
-        $exp = geoPC::getLocalLicenseExpire();
-        if (!defined('DEMO_MODE_TEXT') && $exp != 'never' && $exp != 'pending...' && $exp > 0) {
-            $exp = date('F j, Y', $exp);
-        }
-
-        $tpl_vars['localExpire'] = (defined('DEMO_MODE')) ? 'DEMO' : $exp;
-
-        $exp = geoPC::getLicenseExpire();
-        if (!defined('DEMO_MODE_TEXT') && $exp != 'never' && $exp != 'pending...' && $exp > 0) {
-            $exp = date('F j, Y', $exp);
-        }
-        $tpl_vars['licenseExp'] = (defined('DEMO_MODE')) ? 'DEMO' : $exp;
-
-        $tpl_vars['admin_msgs'] = geoAdmin::m();
-
-        $tpl_vars['show_upgrade_pricing'] = (geoPC::is_ent()) ? false : true;
-        $tpl_vars['maxSeats'] = geoPC::maxSeats();
-        $tpl_vars['maxSeats'] = ($tpl_vars['maxSeats'] == -1) ? 'Unlimited' : $tpl_vars['maxSeats'];
-
-        $view->setBodyTpl('licenseInfo.tpl')
-            ->setBodyVar($tpl_vars);
-    }
-    function update_admin_tools_license()
-    {
-        if (defined('DEMO_MODE')) {
-            geoAdmin::m('No effect on this demo.', geoAdmin::NOTICE);
-            return true;
-        }
-        $onlyData = ($_POST['clearType'] !== 'key');
-        geoPC::clearLicenseKey($onlyData);
-
-        if (!$onlyData) {
-            //log the user out.
-            $session = geoSession::getInstance();
-            $session->logOut();
-            trigger_error('DEBUG LICENSE: LOCAL: License key manually cleared by the admin user.');
-        } else {
-            trigger_error('DEBUG LICENSE: LOCAL: Local license data being manually refreshed by the admin user.');
-        }
-        include GEO_BASE_DIR . 'app_bottom.php';
-        header('Location: index.php?page=admin_tools_license&licenseRefreshed=1');
-        exit;
-    }
 //%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Just a note, when you see these types of things (the `//%%%%...` meant to break up methods), if you want you can remove them.  Same goes for the `// end of bla bla` type comments.  With properly tabbed and organized code there is no need for that type of thing (it is easy to see where something starts/ends), and a lot of times it is incorrect due to copy/pasta.